toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -1,4 +1,8 @@
+acl-2.3.1-2.azl3.aarch64.rpm
+acl-debuginfo-2.3.1-2.azl3.aarch64.rpm
 asciidoc-10.2.0-2.azl3.noarch.rpm
+attr-2.5.2-1.azl3.aarch64.rpm
+attr-debuginfo-2.5.2-1.azl3.aarch64.rpm
```

Comment:
I think this is not that simple. The manifest lists all packages we build through the `toolkit/scripts/toolchain/build_official_toolchain_rpms.sh` script. You'd also need to add these lines into that script before `build_rpm_in_chroot_no_install coreutils`:

```
build_rpm_in_chroot_no_install attr
chroot_and_install_rpms attr-devel
build_rpm_in_chroot_no_install acl

# coretuils requires libattr-devel and libacl-devel
# libacl requires attr
chroot_and_install_rpms  libattr
chroot_and_install_rpms  attr
chroot_and_install_rpms  libacl
(...)
build_rpm_in_chroot_no_install coreutils
```

Please also run a full dev build to make sure all of this actually works.:)

Reply:
I have modified the `build_official_toolchain_rpms.sh` script. Is there something wrong with it? (Were you looking at an update instead of the entire change?)

Reply:
The toolchain itself happily builds: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=620304&view=results

Though, the RPM build seems to be failing for unrelated reasons.

Reply:
Ran a full build and it seems to work:

- x86: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=620642&view=results
- arm: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=620580&view=results

Reply:
I don't know how I missed the script updates, but they looks good to me. Especially now, when I noticed that `libattr-devel` has `Provides: %{name}-devel`, so the BR on `attr-devel` in `acl.spec` is clearer to me.